conditional/blueprints/intro_evals.py

```diff
@@ -108,33 +106,29 @@ def get_fid_cm_count(member_id):
         uid = member.uid
         name = member.cn
         freshman_data = FreshmanEvalData.query.filter(
+            FreshmanEvalData.eval_date > start_of_year(),
             FreshmanEvalData.uid == uid).first()
 
         if freshman_data is None:
             continue
         elif freshman_data.freshman_eval_result != "Pending" and internal:
             continue
 
-        h_meetings = [m.meeting_id for m in
-                      MemberHouseMeetingAttendance.query.filter(
-                          MemberHouseMeetingAttendance.uid == uid
-                      ).filter(
-                          MemberHouseMeetingAttendance.attendance_status == "Absent"
-                      )]
-        member = {
+        h_meetings = [m.meeting_id for m in get_hm(member)]
+        member_info = {
             'name': name,
             'uid': uid,
             'eval_date': freshman_data.eval_date.strftime("%Y-%m-%d"),
             'signatures_missed': freshman_data.signatures_missed,
-            'committee_meetings': get_uid_cm_count(uid),
-            'committee_meetings_passed': get_uid_cm_count(uid) >= 10,
+            'committee_meetings': len(get_cm(member)),
+            'committee_meetings_passed': len(get_cm(member)) >= 10,
```

Comment:
not necessarily for this PR, but we should set 10 to be the number of weeks the candidate is into the process, so that it's technically true to the constitution